src/features/app/App.scss

```diff
@@ -39,7 +39,7 @@
   width: 550px;
   padding-top: 0.5rem;
 
-  @media(max-width: 992px) {
+  @media not (min-width: 992px) {
```

Comment:
Instead of using "not" here, just change min-width to 991px

Reply:
not seemed like more robust way to avoid this issue in the first place?

Reply:
Disagree. Breakpoint is exactly at 992px. No need for a more general case, and I do not find "not min-width" to be intuitive. You're welcome to make the change, or I can fix independently.